pkg/requestcontext/context.go

```diff
@@ -43,7 +50,28 @@ func GetAccessControlContext(ctx context.Context) (*AccessControlContext, error)
 	return nil, nil //nolint: nilnil
 }
 
-// returns either a user has or not rights on 'repository'.
+// returns wether or not the user/anonymous who made the request has read permission on 'repository'.
```

Comment:
```suggestion
// returns whether or not the user/anonymous who made the request has read permission on 'repository'.
```

Same applies to other comments.